src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -280,10 +285,8 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
                 try
                 {
                     // open the input file
-                    using (FileStream fs = new FileStream(InFile, FileMode.Open))
-                    {
-                        SetRequestContent(request, fs);
-                    }
+                    _fileStream = new FileStream(InFile, FileMode.Open);
+                    SetRequestContent(request, _fileStream);
```

Comment:
`StreamContent` does dispose the `Stream` it holds to when it gets disposed, see the code [here](https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/StreamContent.cs#L85)
And `HttpRequestMessage` will dispose the `_content` when it gets disposed, see the code [here](https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs#L203)

So it looks to me that we don't even need to dispose the `filestream` -- it will be automatically disposed when the `HttpRequestMessage` object is garbage collected. However, garbage collector may not kick in soon enough, so we probably should dispose the `HttpRequestMessage` object after using it in a finally block at [here](https://github.com/PowerShell/PowerShell/blob/master/src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs#L383)